package.json

```diff
@@ -59,6 +59,9 @@
       ],
       [
         "@pika/plugin-build-node",
+        {
+          "minNodeVersion": "14"
+        },
```

Comment:
This is wrong. Looks like a bug in Octoherd script

Reply:
yes, it is